packages/visualizations-stories/src/grid/1-pivot-grid.stories.tsx

```diff
@@ -373,13 +373,13 @@ storiesOf("@operational/grid/1. Pivot table", module)
               const h = height - 2 * padding;
```

Comment:
See comment about `h`.

Reply:
```suggestion
              const heightWithoutPadding = height - 2 * padding;
```

It seems really strange that you need these constants so much. May I ask why?